content/2015.12.loadtesting.with.oauth.rst

```diff
@@ -0,0 +1,214 @@
+How to load-test a OAuth-protected HTTP API?
+############################################
+
+:lang: en
+:date: 2015-12-02
+:url: loadtesting-a-http-api-which-uses-oauth-for-authentication
+:summary: This article explains how to test the performance of an HTTP API when its access depends on an external OAuth service.
+
+
```

Comment:
I feel like we miss a section here that briefly explains what is OAuth and how it's used to authenticate to an HTTP API